dotnet/hello_world/Program.cs

```diff
@@ -0,0 +1,210 @@
+            var builder = new ConfigurationBuilder().
+                SetBasePath(Directory.GetCurrentDirectory())
+                .AddJsonFile("appsettings.json", optional: true, reloadOnChange: true);
+
+            IConfigurationRoot configuration = builder.Build();
+
+            _settings = new HelloWorldSettings();
+            configuration.Bind(_settings);
+            #endregion
+
+            HelloWorld helloWorld = new HelloWorld(_settings.ProjectId, _settings.InstanceId);
+            helloWorld.DoHelloWorld();
+            Console.WriteLine("Press any key to exit");
+            Console.ReadKey();
+        }
+
+        public class HelloWorld
+        {
+
+            public string ProjectId { get; set; }
+            public string InstanceId { get; set; }
+
+            public HelloWorld(string projectId, string instanceId)
+            {
+                ProjectId = projectId;
+                InstanceId = instanceId;
+            }
+
+            public void DoHelloWorld()
+            {
+
+                // Build Instance Name
+                InstanceName instanceName = new InstanceName(ProjectId, InstanceId);
+
+                // Build table name Admin
+                Google.Cloud.Bigtable.Admin.V2.TableName _tableAdmin = new Google.Cloud.Bigtable.Admin.V2.TableName(ProjectId, InstanceId, _settings.TableName);
+
+                // Build table name Client
+                Google.Cloud.Bigtable.V2.TableName _tableClient = new Google.Cloud.Bigtable.V2.TableName(ProjectId, InstanceId, _settings.TableName);
+
+                #region 1. Create table with column family
+                // Use try-with-resources to make sure it gets closed
+                try
+                {
+                    BigtableTableAdminClient bigtableTableAdminClient = BigtableTableAdminClient.Create();
+
+                    Console.WriteLine($"Creating new table: {_settings.TableName}");
+                    // Checking if a table with given tableName already exists
+                    var tables = bigtableTableAdminClient.ListTables(instanceName);
+
+                    bool isTableExist = tables.Any(x => x.TableName.TableId == "HelloBigTable");
+
+                    if (!isTableExist)
+                    {
+                        bigtableTableAdminClient.CreateTable(
+                            instanceName,
+                            _settings.TableName,
+                            new Table
+                            {
+                                Granularity = Table.Types.TimestampGranularity.Millis,
+                                ColumnFamilies =
+                                {
+                                    { _settings.ColumnFamily, new ColumnFamily { GcRule = new GcRule { MaxNumVersions = 1 } } }
+                                }
+                            });
+
+                        Console.WriteLine($"Table {_settings.TableName} created succsessfully\n");
+                    }
+                    else
```

Comment:
In general, we put `{` on the same line as `else` in other languages, I think C# is no different.